storage/sql/config.go

```diff
@@ -88,6 +89,13 @@ type Postgres struct {
 	SSL PostgresSSL `json:"ssl" yaml:"ssl"`
 
 	ConnectionTimeout int // Seconds
+
+	// database/sql tunables, see
+	// https://golang.org/pkg/database/sql/#DB.SetConnMaxLifetime and below
+	// using pointers to differentiate "unset" (nil) from 0 (int's zero value)
```

Comment:
Differentiating between zero and nil is hairy. Can we use 0 to set a reasonable default?